benchmarks/apps/ramses/ramses.py

```diff
@@ -0,0 +1,93 @@
+import os.path as path
+import reframe as rfm
+import reframe.utility.sanity as sn
+from benchmarks.modules.utils import SpackTest
+
+
+class Ramses_download_inputs(rfm.RunOnlyRegressionTest):
+
+    descr = 'Download Ramses input files'
+    executable = 'wget'
+    executable_opts = [
+            f'https://zenodo.org/record/7842140/files/data.tgz'  # noqa: E501
+            ]
+    local = True
+
+    @sanity_function
+    def validate_download(self):
+        return sn.assert_true(path.exists('data.tgz'))
+
+
+class RamsesMPI(SpackTest):
+
+    valid_systems = ['*']
+    valid_prog_environs = ['*']
+
+    spack_spec = 'ramses@v1.0.0'
```

Comment:
As far as I can tell, `ramses` and all the other packages below aren't in Spack, so how is this supposed to work?  If you have your own Spack recipes and are willing to keep them public in this repository while the software remain private for the time being, you could add them to our [custom spack repo](https://github.com/ukri-excalibur/excalibur-tests/tree/6f3cfbd4af8c031986e23af9f8da6360dfee36c5/benchmarks/spack/repo).

Reply:
That's the point of this merge request ;) we have added them to the custom spack repo. You can see if you visit our fork / or if you look in the 'files changed' tab. Very happy for them to be public but they cannot be merged into spack as they are private github repos.

![image](https://user-images.githubusercontent.com/25719944/236188032-15973dca-389a-44f0-af00-296e4dfcf7c9.png)

Reply:
Sorry, I completely missed those files at the end :facepalm: